sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -6762,11 +6762,22 @@ ALTER TABLE facility ADD COLUMN areatype varchar(255);
 
 INSERT INTO schema_version (version_number, comment) VALUES (340, 'Add facilities'' address #4027');
 
---2021-02-23 Add event management status to Event #4255
+-- 2021-02-23 Add event management status to Event #4255
 
 ALTER TABLE events ADD COLUMN eventmanagementstatus varchar(255);
 ALTER TABLE events_history ADD COLUMN eventmanagementstatus varchar(255);
 
 INSERT INTO schema_version (version_number, comment) VALUES (341, 'Add event management status to Event #4255');
 
+-- 2021-03-04 Add person email, person phone, testResultVerified to LabMessage #4106
+ALTER TABLE labmessage ADD COLUMN personphone VARCHAR(255);
+ALTER TABLE labmessage ADD COLUMN personemail VARCHAR(255);
+ALTER TABLE labmessage_history ADD COLUMN personphone VARCHAR(255);
+ALTER TABLE labmessage_history ADD COLUMN personemail VARCHAR(255);
+ALTER TABLE labmessage ADD COLUMN testresultverified boolean DEFAULT false;
+ALTER TABLE labmessage_history ADD COLUMN testresultverified boolean DEFAULT false;
```

Comment:
Why are you setting the default constraint? I think SORMAS should not induce data by its own, here.